pkg/kubelet/server.go

```diff
@@ -346,6 +352,18 @@ func (s *Server) InstallDebuggingHandlers() {
 	s.restfulCont.Add(ws)
 }
 
+// InstallApiHandlers registers the HTTP request patterns that serve the kubelet versioned api.
+func (s *Server) InstallApiHandlers() {
+	ws := new(restful.WebService).
+		Path("/api/v1/").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.GET("rawMetrics").To(handlers.RawMetricsHandler(s.host, s.host.GetStatusManager())))
```

Comment:
After a bit more thinking, I think we will have to split this API into pod and node rawMetrics just like the resource metrics API [here](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/compute-resource-metrics-api.md#proposed-endpoints). 
One of the main reasons include adding namespace level access control in the future. 
So in this case, kubelet will expose two separate APIs:
1. `/api/experimental/v1/rawNodeMetrics/` and
2. `/api/experimental/v1/namespaces/<namespace>/rawPodMetrics[/<myPod>]`

To begin with we can support querying all pod raw metrics only. Such a query will be `/api/experimental/v1/namespaces/""/rawPodMetrics`.

We can add support for querying a pod in a specific namespace in a subsequent PR. 

WDYT? @dchen1107